maxim/logger/gemini/client.py

```
@@ -368,28 +536,56 @@ def __init__(self, client: Client, logger: Logger):
+        Args:
+            name: The name of the attribute.
+
+        Returns:
+            Any: The attribute.
+        """
         if name == "_models":
             return self._w_models
         elif name == "_chats":
             return self._w_chats
         elif name == "_aio":
             return self._w_aio
```

Ruff (0.11.9)

Use a dictionary instead of consecutive `if` statements

(SIM116)